docs/usage/user-stories/swissquote.md

```diff
@@ -0,0 +1,229 @@
+We started to collect metrics and store the logs separately for each repository.
+
+Here is the dashboard for our current scheduler:
+
+![A dashboard we made at Swissquote to keep our Renovate runs in check, July 2022](./swissquote_stats.png)
+
+We don’t force any team to use Renovate, each team can decide to opt-in and do it for each project separately.
+
+- 824 repositories enabled out of about 2000 active repositories
+- 8000 PRs were merged since we installed Renovate
+- 239 PRs were merged last month
+- 2 SSDs died on our Renovate machine with the number of projects to clone again and again
+
+## The future of Renovate at Swissquote
+
+Not all teams are using Renovate at this stage, but prefer to manually update their dependencies, we would like to enable Renovate at least for critical dependencies in all repositories and hope to make it useful and easy so they choose by themselves to adopt it for more dependencies.
+
+## How should I get started with Renovate?
+
+If this article convinced you, how should you get started?
+
+1. First, if you know your software is very outdated; don’t enable Renovate right away, you will be swamped with Pull Requests, we’ve been there and it’s not a happy memory. First take the time to **manually upgrade all that you can**. `npm outdated`, `mvn versions:display-dependency-updates` or your package manager’s equivalent of that command can help you get started, test your application and commit that
+1. You can now enable Renovate and will receive a Pull Request to add a configuration file, **read this first PR carefully** as it will explain what kind of PRs you are going to receive and when
+1. Make sure to **pick a schedule**, otherwise you will receive PRs at any time of the day. Our team schedules all third party dependencies on the weekend, automatically merging when the tests pass and we investigate the failing ones on Monday morning
+1. **Group Pull Requests**, if every PR has to go through CI, it can become quite heavy, once most PRs succeed, you can start grouping minor and patch updates so that you get a single PR per repository. Investigating issues becomes a bit trickier because of that. To give you some numbers out of 90 repositories enabled with Renovate, we have on average 4 PRs to investigate per week. Everything else is automatically merged
+1. **Enable auto merging after some time**, make sure your tests are solid and that you won’t upgrade something that wasn’t tested and will break once in production
+
+## Is all this effort worth it?
+
+The short answer is yes !
+
+It took us almost a year, way longer than we thought, to catch up with the latest version of everything in our technical stack.
+But once we were confident enough in our tests to enable automatic merging of pull-requests we were satisfied of the work we did to update our software stack.
```

Comment:
```suggestion
But once we were confident enough in our tests to enable automatic merging of Pull Requests we were satisfied of the work we did to update our software stack.
```

Let's follow the convention on GitHub and just use `Pull Requests`, so without the `-` character.